daprdocs/content/en/developing-applications/building-blocks/observability/sidecar-health.md

```diff
@@ -6,12 +6,18 @@ weight: 5000
 description: Dapr sidecar health checks.
 ---
 
-Dapr provides a way to determine it's health using an HTTP /healthz endpoint.
-With this endpoint, the Dapr process, or sidecar, can be probed for its health and hence determine its readiness and liveness. See [health API ]({{< ref health_api.md >}})
+Dapr provides a way to [determine its health using an HTTP `/healthz` endpoint]({{< ref health_api.md >}}). With this endpoint, the Dapr process, or sidecar, can be:
+
+- Probed for its health
+- Determined for readiness and liveness.
 
 The Dapr `/healthz` endpoint can be used by health probes from the application hosting platform. This topic describes how Dapr integrates with probes from different hosting platforms.
 
-As a user, when deploying Dapr to a hosting platform (for example Kubernetes), the Dapr health endpoint is automatically configured for you. There is nothing you need to configure.
+When deploying Dapr to a hosting platform (for example, Kubernetes), the Dapr health endpoint is automatically configured for you.
+
+{{% alert title="Note" color="primary" %}}
+ Dapr actors also have a health API endpoint where Dapr probes the application for a response to a signal from Dapr that the actor application is healthy and running. See [actor health API]({{< ref "actors_api.md#health-check" >}}).
+{{% /alert %}} 
 
 Note: Dapr actors also have a health API endpoint where Dapr probes the application for a response to a signal from Dapr that the actor application is healthy and running. See [actor health API]({{< ref "actors_api.md#health-check" >}})
```

Comment:
Looks like this note here should be removed. There is now a double (repeated) note.